src/modules/food.js

```diff
@@ -0,0 +1,2 @@
+/* eslint-disable */
```

Comment:
- [ ] Same issue as discussed above, please enable linters and try solving the issues involved.